subprojects/distributions/src/changes/accepted-public-api-changes.json

```diff
@@ -165,6 +165,51 @@
             "type": "org.gradle.api.tasks.testing.AbstractTestTask",
             "member": "Constructor org.gradle.api.tasks.testing.AbstractTestTask()",
             "acceptation": "Extracted super class out of Test which was already stable"
-        }
+        },
+        {
+            "type": "org.gradle.api.tasks.testing.ExecuteTestBuildOperationType",
```

Comment:
❌ These shouldn't be part of the public API, they are only for the scan plugin.

Reply:
moved to internal package